tests/src/test/resources/OpenProjectTestFastpass.conf

```diff
@@ -0,0 +1,29 @@
+include required("OpenProjectTest.conf")
+
+pants.import.targets = ["core/src/main/java"]
```

Comment:
apart from this line this is indentical to Bsp variant. I would do something like this:
create "abstract" OpenProjectTestBsp (or OpenProjectTestFastpass). Create 2 configs that include it without any modifications: OpenProjectTestBspConsole and OpenProjectTestBspUi. Also name tests accordingly. I think it will be more clear than OpenProjectTestBsp and OpenProjectTestFastpass